### PR TITLE
Add workspace to prompt format options

### PR DESCRIPTION
**Pull request by jlee-r7**
# Verification
- [x] `setg Prompt %red%T %grn%J %blu%S %yel%W%clr`
- [x] **Verify** see your workspace in the prompt in yellow
- [x] `workspace -a foo`
- [x] **Verify** see `foo` in the prompt in yellow
- [x] `workspace -d foo`
- [x] **Verify** see `default` in the prompt in yellow

### CODE REVIEW

**Comment by wvu-r7** [comment]
Good addition to #5058. Thanks!

File: lib/rex/ui/text/shell.rb [hunk 1]
```diff
@@ -178,6 +178,9 @@ def run(&block)
           input.prompt.gsub!(/%J/, framework.jobs.length.to_s)
           input.prompt.gsub!(/%L/, Rex::Socket.source_address("50.50.50.50"))
           input.prompt.gsub!(/%D/, ::Dir.getwd)
+          if framework.db && framework.db.workspace
```
Comment (wvu-r7) [inline lib/rex/ui/text/shell.rb]: This probably should have been ```framework.db.active```.

Comment (wvu-r7) [inline lib/rex/ui/text/shell.rb]: Fixed in https://github.com/rapid7/metasploit-framework/commit/5f8d58f2140eabad7b674bea40ab709d63fad1d9.

**Comment by wvu-r7** [comment]
https://github.com/rapid7/metasploit-framework/commit/5f8d58f2140eabad7b674bea40ab709d63fad1d9

**Comment by mubix** [comment]
Love this!

**Comment by todb-r7** [comment]
Why not make this default?

**Pull request merged**